package.json

```diff
@@ -6,6 +6,7 @@
   "dependencies": {
     "ajv": "^8.1.0",
     "cookie": "^0.4.0",
+    "fastify": "^3.20.2",
```

Comment:
Unfortunately you should not be using fastify here :(. See the other tests, you can do this with a plain Node.js server.

Reply:
`

          test('value of request url when using inject should not differ', async (t) => {
          t.plan(1)
          const ap = fastify();
          ap.get("*", function (request, reply) {
            reply.send({ url: request.url });
          });
          const res = await ap.inject({
            method: "GET",
            url: "http://example.com:8080//foo",
          });
          let v=JSON.parse((res.body))  //added this Json parse
        
          t.equal(v.url,'//foo')
          
        })
`

The code above works fine but as you said I removed fastify and did it with express. Now express doesn't have function inject so:
`

            test('value of request url when using inject should not differ', async (t) => {
              t.plan(1)
            
              const app = express()
            
              app.get('*', (req, res) => {
                console.log(req.url)
                res.jsonp({url:req.url})
              })
            
              inject(app, { method: 'GET', url: 'http://example.com:8080//hello' }, (err, res) => {
                console.log(res)
                t.equal(res.body, '')
              })
            })

`

But that gives an error and somehow the res.body is empty. Do you have any idea?

Reply:
Do not use express, just use http.createServer().

Reply:
Hey did that, it's working could you review